.github/workflows/ci.yml

```diff
@@ -49,10 +52,10 @@ jobs:
       shell: powershell
       # See https://github.com/msys2/MSYS2-packages/issues/258
       run: |
-        C:\msys64\usr\bin\rm -rf $env:MINGW_PREFIX/bin/libcrypto-3-x64.dll $env:MINGW_PREFIX/bin/libssl-3-x64.dll $env:MINGW_PREFIX/etc/ssl/openssl.cnf
-        C:\msys64\usr\bin\pacman --noconfirm --ask 20 --sync --refresh --refresh --sysupgrade --sysupgrade
-        Get-Process | Where-Object {$_.path -like 'C:\msys64*'} | Stop-Process
-        C:\msys64\usr\bin\pacman --noconfirm --sync --refresh --refresh --sysupgrade --sysupgrade
+        iex "${env:RI_DEVKIT}/usr/bin/rm -rf $env:MINGW_PREFIX/bin/libcrypto-3-x64.dll $env:MINGW_PREFIX/bin/libssl-3-x64.dll $env:MINGW_PREFIX/etc/ssl/openssl.cnf"
```

Comment:
Not sure what's the reason that we need to remove these two files. Also the filename in arm64 do not have `-x64`.